react-native/navigation/src/app/screens/TabRouter/screens/Posts/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import {
+  View,
+  Text,
+  FlatList,
+  TouchableHighlight
+} from 'react-native';
+
+import data from '../../../../../mock';
+import routes from '../../../../../constants/routes';
+import UserAvatar from '../../../../components/UserAvatar';
+
+import Header from './header';
+import styles from './styles'
+
+class Posts extends Component {
+  static navigationOptions = ({ navigation }) => ({
+    title: 'Posts',
+    tabBarLabel: 'Posts',
+    headerLeft: null,
+    header: <Header />
```

Comment:
I think `headerLeft: null` isn't necessary since `header` overrides both `Left` and `Right` headers